serde_derive_internals/Cargo.toml

```diff
@@ -12,8 +12,16 @@ readme = "README.md"
 include = ["Cargo.toml", "src/**/*.rs", "README.md", "LICENSE-APACHE", "LICENSE-MIT"]
 
 [dependencies]
-syn = { version = "0.11.10", default-features = false, features = ["parsing"] }
-synom = "0.11"
+synom = { git = 'https://github.com/dtolnay/syn' }
+literalext = { git = 'https://github.com/mystor/literalext' }
+
+[dependencies.syn]
+git = 'https://github.com/dtolnay/syn'
+default-features = false
+features = ["parsing", "extra-traits"]
 
 [badges]
 travis-ci = { repository = "serde-rs/serde" }
+
+[features]
+extra-traits = ["syn/extra-traits"]
```

Comment:
extra-traits is listed twice, both as a required and optional feature. I think it should only be listed in one of those two places.